### ISSUE TITLE
Modal focus trap not working if no focusable element exists after modal

### ISSUE THREAD

**Issue by justindmyers** [bootstrap-vue/bootstrap-vue#1615]
The modal focus trap works fine if there are focusable elements after the modal but if the modal itself is last, the browser receives focus instead.

It looks as if `content.focus()` only works if the focus is still within the page. In this case, the `onFocusout` event stops working as it's fired after the browser has already received focus so we can't programmatically force focus back on the modal.

Chrome, Firefox, and Edge all exhibit this behavior.

Example: https://codepen.io/justindmyers/full/GQMqOR/
